java-jsp/src/test/java/org/sonar/java/jsp/JasperTest.java

```diff
@@ -133,10 +137,24 @@ void test_source_map() throws Exception {
     assertThat(generatedFile.sourceMap()).isNotNull();
   }
 
+  @Test
+  void test_failure_invalid_classpath() throws Exception {
```

Comment:
I am unsure about what this test actually verifies.

Reply:
renamed to make it a bit clear, also changed the method which is mocked so it's more realistic